loopsPartTwo.js

```diff
@@ -221,8 +232,36 @@ console.log(compareArray(routineThree, ["situp", "situp", "situp"]));
 
 function calculateRoutineDifficulty(routine) {
   let difficulty = "";
-  // Your Code Here!
-  return difficulty;
+  let situpPointValue = 1;
+  let legraisePointValue = 2;
+  let pushupPointValue = 4;
+  let pullupPointValue = 10;
+  let totalPoints = 0;
+  for (index = 0; index < routine.length; index++) {
+    if (routine[index] === "situp") {
+      routine[index] = situpPointValue;
+      totalPoints = situpPointValue + totalPoints;
+    } else if (routine[index] === "legraise") {
+      routine[index] = legraisePointValue;
+      totalPoints = legraisePointValue + totalPoints;
+    } else if (routine[index] === "pushup") {
+      routine[index] = pushupPointValue;
+      totalPoints = pushupPointValue + totalPoints;
+    } else if (routine[index] === "pullup") {
+      routine[index] = pullupPointValue;
+      totalPoints = pullupPointValue + totalPoints;
+    }
+  }
+  if (totalPoints < 30) {
+    let difficulty = "Easy";
+    return difficulty;
```

Comment:
instead putting `return difficulty` in each of these conditional blocks, you can just return `difficulty` at the end of all of the conditionals